mcpjam-inspector/client/src/components/score/score-site.css

```
@@ -0,0 +1,49 @@
+  --score-bg: #000000;
+  --score-fg: #f4f1ea;
+  --score-muted: #9a9690;
+  --score-primary: oklch(68.3% 0.138 38.7);
+  --score-primary-fg: #1a0c04;
+  --score-surface: #0c0c0c;
+  --score-border: #2a2826;
+  --score-card: #090504d1;
+  --font-score-display: "Montserrat", sans-serif;
+  --font-score-sans: "Instrument Sans", sans-serif;
+  --font-score-mono: "JetBrains Mono", monospace;
+  color: var(--score-fg);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)